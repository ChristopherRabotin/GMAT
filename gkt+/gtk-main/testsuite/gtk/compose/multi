<Multi_key> <s> <e> <q> : "!"
<Multi_key> <u> <b> <2> <3> : "/"
<Multi_key> <s> <a> <s> : "_"
