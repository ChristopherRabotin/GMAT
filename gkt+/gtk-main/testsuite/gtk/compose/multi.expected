# n_sequences: 3
# max_seq_len: 5
# n_index_size: 1
# data_size: 19
# n_chars: 0
<Uff20> <U73> <U61> <U73> : "_" # U5f
<Uff20> <U73> <U65> <U71> : "!" # U21
<Uff20> <U75> <U62> <U32> <U33> : "/" # U2f
